nixos/modules/profiles/perlless.nix

```diff
@@ -1,6 +1,5 @@
-# WARNING: If you enable this profile, you will NOT be able to switch to a new
-# configuration and thus you will not be able to rebuild your system with
-# nixos-rebuild!
+# WARNING: If you enable this profile, you will NOT be able to use nixos-rebuild!
```

Comment:
With stc-ng you will actually be able to use nixos-rebuild because nixos-rebuild is just a bash script! So you could remove the entire comment.

Reply:
nixos-rebuild is removed because of `system.disableInstallerTools = lib.mkDefault true;` so maybe a different new warning?